app/src/sm_at_host.c

```diff
@@ -712,42 +668,47 @@ static size_t null_handler(const uint8_t *buf, const size_t len)
 	return processed;
 }
 
-void sm_at_receive(const uint8_t *buf, size_t len)
+size_t sm_at_receive(const uint8_t *buf, size_t len, bool *stop_at_receive)
 {
 	size_t ret = 0;
 
+	*stop_at_receive = false;
```

Comment:
This is set in three places plus in the caller of this function. Not a big harm but maybe a bit unnecessary too. Not sure how compiler optimizes these.